sdk/program/src/message/v0.rs

```diff
@@ -0,0 +1,385 @@
+#![allow(clippy::integer_arithmetic)]
+
+use super::MessageHeader;
+use crate::{
+    hash::Hash,
+    instruction::CompiledInstruction,
+    pubkey::Pubkey,
+    sanitize::{Sanitize, SanitizeError},
+    short_vec,
+};
+
+/// Alternative version of `Message` that supports succinct account loading
+/// through an on-chain address map.
+#[frozen_abi(digest = "FqrtpvgZZxJBCqcETJ7LYVdaLia2n4izZ813DxgyrW2u")]
```

Comment:
this can be removed and should be covered with new `frozen_abi()` on `MessageVersions`: for https://github.com/solana-labs/solana/pull/18725/files#diff-d4c52a4504e9e02f6f13335ff68c21b5dadccc0e2ac44435e74c86dc2c1f8b30R30